lib/histogram.js

```diff
@@ -64,6 +65,7 @@ class Histogram {
 
 		this.name = config.name;
 		this.help = config.help;
+		this.aggregate = config.aggregator || AGGREGATORS.SUM;
```

Comment:
Couldn't you set this in the object assign step instead? Will be easier to see all the defaults if they are set on the same place